benchmarks/run-benchmarks.sh

```diff
@@ -0,0 +1,265 @@
+  echo "It assumes you have the following things installed on your machine:"
+  echo "  - git (https://github.com/git-guides/install-git)"
+  echo "  - jbang (https://www.jbang.dev/download)"
+  echo "  - jq (https://stedolan.github.io/jq)"
+  echo
+  echo "Syntax: run-benchmarks.sh [options]"
+  echo "options:"
+  echo "  -b <SCM_REPO_BRANCH>                The branch in the SCM repo"
+  echo "                                          Default: 'main'"
+  echo "  -c <CGROUPS_CPUs>                   Constrain to certain CPUs via cgroups"
+  echo "                                          NOTE: Can be a comma-separated list (i.e. '0,2,4,6,8,10,12,14')"
+  echo "  -d                                  Purge/drop OS filesystem caches between iterations"
+  echo "  -g <GRAALVM_VERSION>                The GraalVM version to use (from SDKMAN)"
+  echo "                                          Default: 25-graalce"
+  echo "  -h <HOST>                           The HOST to run the benchmarks on"
+  echo "                                          LOCAL is a keyword that can be used to run everything on the local machine"
+  echo "                                          NOTE: In the future we may have a container image that will spin up automatically"
+  echo "                                          Default: LOCAL"
+  echo "  -i <ITERATIONS>                     The number of iterations to run each test"
+  echo "                                          Default: 3"
+  echo "  -j <JAVA_VERSION>                   The Java version to use (from SDKMAN)"
+  echo "                                          Default: 25-tem"
+  echo "  -l <SCM_REPO_URL>                   The SCM repo url"
+  echo "                                          Default: 'https://github.com/quarkusio/spring-quarkus-perf-comparison.git'"
+  echo "  -m <CGROUPS_MAX_MEMORY>             Constrain available memory via cgroups"
+  echo "                                          Default: 14G"
+  echo "  -n <NATIVE_QUARKUS_BUILD_OPTIONS>   Native build options to be passed to Quarkus native build process"
+  echo "  -o <NATIVE_SPRING_BUILD_OPTIONS>    Native build options to be passed to Spring native build process"
+  echo "  -p <PROFILER>                       Enable profiling with async profiler"
+  echo "                                          Accepted values: none, jfr, flamegraph"
+  echo "                                          Default: none"
+  echo "  -q <QUARKUS_VERSION>                The Quarkus version to use"
+  echo "  -r <RUNTIMES>                       The runtimes to test"
+  echo "                                          Accepted values (1 or more of): quarkus3-jvm, quarkus3-native, spring3-jvm, spring3-jvm-aot, spring3-native"
+  echo "                                          Default: 'quarkus3-jvm,quarkus3-native,spring3-jvm,spring3-jvm-aot,spring3-native'"
+  echo "  -s <SPRING_BOOT_VERSION>            The Spring Boot version to use"
```

Comment:
Ditto as https://github.com/quarkusio/spring-quarkus-perf-comparison/pull/15/files#r2443753270

Not specifying both is clearly not cool, since it means nothing has to run.
But generally speaking it makes super simple to try the script even without any comparison in mind e.g. 

```bash
./run-benchmarks.sh  -q 3.28.4
```
will just work as it is

Reply:
I've updated the script docs